packages/components/SelectionDropdown.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useRef } from "react";
```

Comment:
Seems that we has already the Dropdown component, please double check

Reply:
I checked, but it didn't work for me, so I created a component.